debian/rules

```diff
@@ -150,6 +150,8 @@ override_dh_python2:
 	dh_python2 -p ceph-common
 	dh_python2 -p ceph-base
 	dh_python2 -p ceph-osd
+	dh_python2 -p ceph-mon
+	dh_python2 -p ceph-mgr
 
 override_dh_python3:
 	for binding in rados cephfs rbd; do                                \
```

Comment:
Added another commit. I don't have any Debian system to test this on, unfortunately.